ocs_ci/ocs/platform_nodes.py

```diff
@@ -59,6 +60,15 @@ def get_nodes_platform(self):
         return self.cls_map[platform]()
 
 
+class RHVNodes(object):
```

Comment:
For better ordering, let's place this class below the base class of `NodesBase`

Reply:
@ebenahar done, have moved RHVNodes class below the class of NodesBase